### DISCUSSION TITLE
.deb file legit?

### DISCUSSION THREAD

**Discussion by balassb** [allure-framework/allure2#1603] [General Discussion]
Hi there,
I've downloaded the latest .deb file (allure_2.17.3-1_all.deb) and after installing it, I got this hacky info, that is worrying more than funny. What's this about? Should I be concerned? Is this expected?
![image](https://user-images.githubusercontent.com/100790585/156406315-fda9e10f-df31-4807-ab34-de8c4714d202.png)
Reads like:
Allure of the Stars
Explore an abandoned spaceship with a party of adventurers.
Allure of the Stars is a near-future Sci-Fi roguelike and tactical squad game....

**Comment by baev** [comment]
looks like our package name ('allure') conflicts with this one. Thanks for mentioning, we'll take a look
